### DISCUSSION TITLE
Looking for a slider-like widget for input and output.

### DISCUSSION THREAD

**Discussion by Eirikur** [Textualize/textual#2362] [Ideas]
I'm prototyping something that would probably use a USB knob box, but
thought I might be able to use textual for prototyping. I didn't find any analog-ish widgets for updating or dragging. I saw that progress bars are on the roadmap. I could probably use a progress bar if it can go backward.

**Comment by davep** [comment]
Progress bars are being worked on right now, so won't be too far off. There isn't a slider in the library at the moment, as you'll have noticed, and I'm not aware of any third party library that provides them right now. I do seem to recall someone showing off in the `#show-and-tell` channel in [our Discord server](https://discord.gg/Enf6Z3qhVr) an implementation they were working on; might be worth having a search back through there and seeing if they made it public.
